.github/workflows/charts-lint-test.yaml

```diff
@@ -111,7 +111,7 @@ jobs:
       if: needs.lint.outputs.changed == 'true' || needs.lint.outputs.common == 'true'
 
     - name: Run chart-testing (install)
-      run: ct install --config .github/ct.yaml --excluded-charts ""
+      run: ct install --config .github/ct.yaml --excluded-charts "dnsmadeeasy-webhook"
```

Comment:
I am not sure if we want to do this because it skips all linting. But we really don't have a standard way with dealing with Helm charts that fail to install.

Reply:
Ideally instead of disabling the test we would have a way to install the operator chart prior to the to-be-tested chart.

Should we keep like this for now and open an issue to pre-install depencies? Should we move the exclude list into a dedicated file?

Reply:
In the meanwhile we have the split